package.json

```diff
@@ -46,8 +46,9 @@
   "scripts": {
     "test": "vitest",
     "test:watch": "yarn test -- -w",
-    "build": "rollup -c",
+    "build": "yarn build:icons && rollup -c",
```

Comment:
Not sure why you need to rebuild the icons on every build, they are pretty static, you can just keep the icon components and remove the svgs

Reply:
basically you are right, but I wanted to do it in a way where I do have the svgs origin in case I'll need them + they are not included in my `dist`.
also, to make it easier for future icons

Reply:
The svgs are included in the react components, it just wraps them in a component. But anyway it's not critical, just feels redundant a bit

Reply:
@talkor I've removed the svgs and added in the README how you should add a new icon to the repo